practice_2/NewsApp/View/DetailsNewsViewController.swift

```diff
@@ -0,0 +1,48 @@
+import UIKit
+
+class DetailsNewsViewController: UIViewController {
+    @IBOutlet private weak var newsImage: UIImageView!
+    @IBOutlet private weak var newsDescriptionTextView: UITextView!
+
+    var article: TopArticle?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configure()
+    }
+
+    private func configure() {
+        guard let article = article else { return }
+        if let url = URL(string: article.urlToImage) {
+            newsImage.load(url: url)
+        } else {
+            print("Could not create URL object for \(article.urlToImage)")
+        }
+        newsDescriptionTextView.text = article.description
+    }
+
+    private func getNewsUrl() -> URL? {
+        if let article = article {
+            return URL(string: article.url)
+        }
+        return URL(string: "https://www.google.com")
```

Comment:
it's better to show some alert or warning, just not to confuse a user

Reply:
I agree with you. Fixed. Thanks.